scenarios/steadystate.profile.yml

```
@@ -0,0 +1,27 @@
+# This file adds a new "Steady State Working Set" result (working-set/steady) to the "application" job.
+# It averages all the working-set measurements after the warmup is done.
+
+results:
+
+- name: working-set/steady
+  description: Steady State Working Set (MB)
+  format: n0
+
+onResultsCreated:
+  - |
+    // This measures the working set value based only on the time when the warmup has ended
+
+    let duration = benchmarks.jobs.load.variables.duration;
+    let measurements = benchmarks.jobs.application.measurements[0].filter(m => m.name == "runtime-counter/working-set");
```

Can this be 

```suggestion
    let measurements = benchmarks.jobs.application.measurements[0].filter(m => m.name =="benchmarks/working-set");
```

instead? That way it will work on non-.NET apps as well.